Web.md

```diff
@@ -124,6 +124,81 @@ MIME type mismatch or `opaque` response types
 [reject](http://tc39.github.io/ecma262/#sec-rejectpromise) the Promise with a
 `WebAssembly.CompileError`.
 
+### Structured Clone of a `WebAssembly.Memory`
+
+A `WebAssembly.Memory` is under some circumstances a
+[cloneable object](https://html.spec.whatwg.org/multipage/infrastructure.html#cloneable-objects), and will impact the
+[HTML Standard](https://html.spec.whatwg.org/multipage/infrastructure.html#serialization-steps) as follows:
+
+[StructuredSerializeInternal](https://html.spec.whatwg.org/multipage/infrastructure.html#structuredserializeinternal) adds a new step after step 12:
+
+* Otherwise, if `value` has an [[WebAssembly.Memory]] internal slot, then:
+  1. Let `size` be value.[[BufferObject].
+  2. If !  [IsSharedArrayBuffer](https://tc39.github.io/ecma262/#sec-issharedarraybuffer) is true, then:
+    1. If `forStorage` is true, then throw a
+       ["DataCloneError" DomException](https://heycam.github.io/webidl/#datacloneerror).
+    2. Set `serialized` to { [[Type]]: "WebAssembly.Memory",
+       [[ArrayBufferData]]: value.[[BufferObject]].[[ArrayBufferData],
+       [[ArrayBufferByteLength]]: size,
+       [[AgentCluster]]: the [current Realm Record](https://tc39.github.io/ecma262/#current-realm)'s corresponding [agent cluster](https://tc39.github.io/ecma262/#sec-agent-clusters) }.
+    3. Otherwise:
+       1. Let `dataCopy` be [CreateByteDataBlock](https://tc39.github.io/ecma262/#sec-createbytedatablock)(size).
+          NOTE: This can throw a RangeError exception on allocation failure.
+       2. Perform `CopyDataBlockBytes`(https://tc39.github.io/ecma262/#sec-copydatablockbytes)
+          (dataCopy, 0,
+          value.[[BufferObject]].[[ArrayBufferData]], 0, size).
+       3. Set `serialized` to { [[Type]]: "WebAssembly.Memory",
+          [[ArrayBufferData]]: dataCopy, [[ArrayBufferByteLength]]: size }.
+
```

Comment:
What if it's the reverse of 2. above with the weird bang? Doesn't seem to have a sequel for the opposite.